### ISSUE TITLE
update feature/coupled-crow to use latest ufs-weather-model 

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#324]
To update feature/coupled-crow to use the latest develop branch of ufs-weather-model, we need to track the changes in the tests/parm folder in the diffs here: 
https://github.com/ufs-community/ufs-weather-model/compare/Prototype-6.0beta...develop

Updates are needed to: 
* change name of mediator in nems.configure 
* MOM_input has new option ALLOW_LANDMASK_CHANGES (we use the default value which is false, but still should keep in sync)
* ice_in has changes
* the build needs to be updated as well (see https://github.com/JessicaMeixner-NOAA/global-workflow/commit/cdc04db78d946d904f4e1235956e48864c9a3ba3 for needed change)
* use ocean mesh file (a new ocean mesh fix file is not needed as its the same as the ice mesh file)

**Comment by JessicaMeixner-NOAA** [comment]
Another item: 
* After PR https://github.com/ufs-community/ufs-weather-model/pull/586  this line https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/ush/nems_configure.sh#L158 needs to be changed from: 
`cp $HOMEgfs/sorc/ufs_coupled.fd/CMEPS-interface/CMEPS/mediator/fd_nems.yaml fd_nems.yaml
`
to 
`cp $HOMEgfs/sorc/ufs_coupled.fd/tests/parm/fd_nems.yaml fd_nems.yaml`

**Comment by JessicaMeixner-NOAA** [comment]
The mesh ocn/ice are the same: 
  mesh_ocn = mesh.mx100.nc
same as   mesh_ice = mesh.mx100.nc

**Comment by WalterKolczynski-NOAA** [comment]
The ufs module naming convention has been changed as well.

**Comment by WalterKolczynski-NOAA** [comment]
I'll take care of this after we clear out some of the existing PRs.

**Comment by JessicaMeixner-NOAA** [comment]
With the latest merge into ufs-weather-model today (6/11), https://github.com/NOAA-EMC/global-workflow/issues/325 is also needed when updating to the latest.

**Comment by WalterKolczynski-NOAA** [comment]
I think I've got all of these except the first thing. I don't see any mediator name changes in the current UFS develop nems.configure templates.

**Comment by WalterKolczynski-NOAA** [comment]
So I think I am just waiting on #325 now so I can test?

**Comment by JessicaMeixner-NOAA** [comment]
> So I think I am just waiting on #325 now so I can test?

@WalterKolczynski-NOAA here's a branch with the updates: https://github.com/jessicameixner-noaa/global-workflow/tree/coupled/updatewavemultiinp 
However, I couldn't test them due to a lmod issue on hera, I'll try to test this on orion, but since it's going to take a while I thought I'd just point you to this for now.

**Comment by JessicaMeixner-NOAA** [comment]
> I think I've got all of these except the first thing. I don't see any mediator name changes in the current UFS develop nems.configure templates.

Comparing the files https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/ush/nems.configure.cpld_wave.IN and https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/nems.configure.cpld_wave.IN 
and/or comparing 
https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/ush/nems.configure.cpld.IN and https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/nems.configure.cpld.IN

The diffs are more/less: 
diff nems.configure.cpld_wave.IN /scratch2/NCEPDEV/climate/Jessica.Meixner/p7waveup/ufs-weather-model/tests/parm/nems.configure.cpld_wave.IN
```
21c21
<   DumpFields = @[DumpFields]
---
>   DumpFields = false
31c31
<   DumpFields = @[DumpFields]
---
>   DumpFields = false
33a34
>   mesh_ocn = @[MESHOCN_ICE]
41c42
<   DumpFields = @[DumpFields]
---
>   DumpFields = false
44c45
<   mesh_ice = @[MESHICE]
---
>   mesh_ice = @[MESHOCN_ICE]
98c99
<       history_n = 0
---
>       history_n = 1
115a117
>       use_mommesh = @[use_mommesh]
```

We can ignore the DumpFields differences as the workflow has a variable configuration for that instead of just setting it to false.   The differences of note that need to be updated are: 

```
33a34
>   mesh_ocn = @[MESHOCN_ICE]
44c45
<   mesh_ice = @[MESHICE]
---
>   mesh_ice = @[MESHOCN_ICE]
101a103
>       use_mommesh = @[use_mommesh]
```

for the additional configuration of mesh_ocn and use_mommesh and then the change from MESHICE-> MESHOCN_ICE because this is the same mesh file so it should be the same for all.

**Comment by WalterKolczynski-NOAA** [comment]
I've got all those already. I thought #325 was some else.

**Comment by JessicaMeixner-NOAA** [comment]
> I've got all those already. I thought #325 was some else.

Perfect, thanks! Nothing else is needed other than adding the extra "F F" and "no" "no" to the appropriate lines for creating ww3_multi.inp

**Comment by WalterKolczynski-NOAA** [comment]
Do I need to update any locations for new fix files?

**Comment by JessicaMeixner-NOAA** [comment]
> Do I need to update any locations for new fix files?

for the waves? no 
For using the new tile oro data and other physics options? yes, but I'd consider both of those separate items/issues

**Comment by WalterKolczynski-NOAA** [comment]
Okay, I wasn't sure if those were required for the updated model. Setting up test now.

**Comment by WalterKolczynski-NOAA** [comment]
> I think I've got all of these except the first thing. I don't see any mediator name changes in the current UFS develop nems.configure templates.

I found the change once my tests failed.

**Comment by WalterKolczynski-NOAA** [comment]
On hold waiting for ufs-community/ufs-weather-model/pull/664 to fix CICE output.

**Comment by JessicaMeixner-NOAA** [comment]
The PR ufs-community/ufs-weather-model/pull/664  was scheduled for 7/7
